pkg/asset/installconfig/aws/permissions.go

```diff
@@ -118,6 +118,7 @@ var permissions = map[PermissionGroup][]string{
 		"iam:GetRole",
 		"iam:GetRolePolicy",
 		"iam:GetUser",
+		"iam:ListAttachedRolePolicies",
```

Comment:
This should go in `PermissionDeleteBase` instead of `PermissionCreateBase`.

Reply:
Updated.